### PR TITLE
Add a context menu to the credential items

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
[Add credential context menu](https://trello.com/c/Ttn6awTC/271-add-credential-context-menu)

**Comment by huwshimi** [comment]
QA:
- view the new profile
- open the list of credentials
- click on the ...
- the menu should open (but the items currently do nothing)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8463/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8464/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8467/

File: jujugui/static/gui/src/app/components/more-menu/more-menu.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ class MoreMenu extends React.Component {
           role="button"
           tabIndex="0">
           <SvgIcon
-            name="contextual-menu-16"
+            name={this.props.icon || 'contextual-menu-16'}
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/more-menu/more-menu.js]: Is this so that elsewhere we still use the old icon?

**Comment by hatched** [comment]
Looks good @huwshimi although I had to `clean-all` to get the new svg asset. So that'll need to be communicated to others once this lands.

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**